packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Text, useOnClickOutside, Modal, Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useStore, observer } from '@deriv/stores';
+import './wallets-migration-failed.scss';
+
+const WalletsMigrationFailed = observer(() => {
+    const { traders_hub, ui } = useStore();
+    const { is_wallet_migration_failed, setWalletsMigrationFailedPopup } = traders_hub;
+    const { is_mobile } = ui;
+
+    const mobile_add_class = is_mobile ? '--mobile' : '';
+
+    const wallets_migration_failed_ref = React.useRef<HTMLDivElement>(null);
+
+    const handleClose = () => {
+        setWalletsMigrationFailedPopup(false);
+    };
+
+    const handLiveChatButtonClick = () => {
+        window.LC_API?.open_chat_window();
+        setWalletsMigrationFailedPopup(false);
+    };
+
+    const validateClickOutside = (e: MouseEvent) => {
+        return is_wallet_migration_failed && !wallets_migration_failed_ref?.current?.contains(e.target as Node);
+    };
+
+    useOnClickOutside(wallets_migration_failed_ref, handleClose, validateClickOutside);
+
+    return (
+        <Modal
+            is_open={is_wallet_migration_failed}
+            toggleModal={handleClose}
+            width={is_mobile ? '32.3rem' : '44rem'}
+            should_header_stick_body={false}
+            has_close_icon={false}
+        >
+            <div ref={wallets_migration_failed_ref}>
+                <Modal.Body className={`wallets-migration-failed${mobile_add_class}`}>
```

Comment:
i think we could have used the classNames function for this tbh, something like this?

```suggestion
                <Modal.Body className={classNames('wallets-migration-failed', {
                       'wallets-migration-failed--mobile': is_mobile,
                })}>
```